DAO.sol

```diff
@@ -821,7 +821,8 @@ contract DAO is DAOInterface, Token, TokenCreation {
 
 
     function halveMinQuorum() returns (bool _success) {
-        if (lastTimeMinQuorumMet < (now - quorumHalvingPeriod)) {
+        if ((lastTimeMinQuorumMet < (now - quorumHalvingPeriod) || msg.sender == curator)
+			&& lastTimeMinQuorumMet < (now + minProposalDebatePeriod)) {
```

Comment:
Okay I see. Write that in a comment there since that can't be guessed from the code.

Also if the changes of this PR do make it in the code we would also need to change the whitepaper to include them since they are pretty important.